samples/unit-tests/series-columnpyramid/basic/demo.js

```diff
@@ -54,3 +54,39 @@ QUnit.test('Column pyramid series - 0 dataLabel #12514', function (assert) {
         'Label should be translated and had different position translate attributes than starting values #12514'
     );
 });
+
+QUnit.test(
```

Comment:
This test is good, but in order to keep the processing time down in our test suite, it can be combined with an existing test. The first test in this file ('Column pyramid series') can be extended by setting an `yAxis.width`.